### PR TITLE
Features/7735

### PR DESCRIPTION
**Pull request by sslaws**
Completes work on [AB#7735](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/7735)

### CODE REVIEW

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_276)

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_276)

**Comment by healthgatewaybc** [comment]
Medication Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_276)

**Comment by healthgatewaybc** [comment]
Patient Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=patientservice_pr_276)

File: Apps/WebClient/src/Server/Startup.cs [hunk 1]
```diff
@@ -43,6 +43,7 @@ public class Startup
     {
         private readonly StartupConfiguration startupConfig;
         private readonly IConfiguration configuration;
+        private readonly ILogger logger;
 
         private ILogger Logger { get; set; }
```
Comment (TiagoGraf89) [inline Apps/WebClient/src/Server/Startup.cs]: here

**Comment by healthgatewaybc** [comment]
Immunization Project [SonarQube Report](https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=immunization_pr_276)

**Pull request merged**